<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Record Mode</string>
  </property>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="uiRecordModePanes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>801</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="uiRegularPane">
    <widget class="Label" name="uiRegularLengthInSeconds">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>%d seconds</string>
     </property>
    </widget>
    <widget class="Label" name="uiRegularLengthInFrames">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>%d frames</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Record Length</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>10</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black; border-left-width: 0px;</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Segments</string>
     </property>
    </widget>
    <widget class="Label" name="uiRegularLengthNumSegments">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="uiSegmentedPane">
    <widget class="HeaderLabel" name="headerlabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Record Length</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Segments</string>
     </property>
    </widget>
    <widget class="SpinBox" name="uiSegmentLengthInFrames">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="clickMarginTopSetter" stdset="0">
      <enum>SpinBox::half</enum>
     </property>
    </widget>
    <widget class="DecimalSpinBox" name="uiSegmentLengthInSeconds">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="clickMarginBottomSetter" stdset="0">
      <enum>DecimalSpinBox::half</enum>
     </property>
    </widget>
    <widget class="SpinBox" name="uiSegmentLengthNumSegments">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="Label" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>sec.</string>
     </property>
    </widget>
    <widget class="Label" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>frames</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>10</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black; border-left-width: 0px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="uiRunAndGunPane">
    <widget class="Label" name="uiRunNGunTimeInFrames">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>133213 frames in total</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Segments</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Record Length</string>
     </property>
    </widget>
    <widget class="Label" name="uiRunNGunTimeInSeconds">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1000 seconds in total</string>
     </property>
    </widget>
    <widget class="Label" name="uiRunNGunNumSegments">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Many</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>10</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black; border-left-width: 0px;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="HeaderLabel" name="headerlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Record Mode</string>
   </property>
  </widget>
  <widget class="ComboBox" name="uiRecordMode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>661</width>
     <height>81</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Regular: Record frames in one buffer, using the all available RAM.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Segmented: Record frames in separate buffers, one buffer per trigger.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Run &amp; Gun: Clips are recorded into buffers until RAM is exhausted.</string>
    </property>
   </item>
  </widget>
  <widget class="Button" name="uiDone">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combo_box</header>
  </customwidget>
  <customwidget>
   <class>DecimalSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>decimal_spin_box</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>header_label</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spin_box</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiDone</tabstop>
  <tabstop>uiRecordMode</tabstop>
  <tabstop>uiSegmentLengthInSeconds</tabstop>
  <tabstop>uiSegmentLengthNumSegments</tabstop>
  <tabstop>uiSegmentLengthInFrames</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
